index.rst

```diff
@@ -212,7 +212,10 @@ filesystem directory.
 Files are read top to bottom and the most recent rules found take
 precedence. If multiple EditorConfig files have matching sections, the rules
 from the closer EditorConfig file are read last, so pairs in closer
-files take precedence.
+files take precedence. Thus, in case a given file matches multiple sections
+within a single ``.editorconfig`` file, the rules defined in the section that
+comes later in the ``.editorconfig`` file take precedence over rules defined
+in the section that comes earlier in the same ``.editorconfig`` file.
```

Comment:
They look like should be put right after the first period of the paragraph. You can separate "If multiple EditorConfig ..." to a new paragraph.

Also, please change the word "rule" with "Pair". "Rule" is undefined but "Pair" is defined above.

Reply:
Done @xuhdev